### ISSUE TITLE
Tepl (and therefore Gedit) not working on macOS Monterey.

### ISSUE THREAD

**Issue by Baccio26** [Homebrew/homebrew-core#88743]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
HOMEBREW_VERSION: 3.3.2-46-gde5a13b
ORIGIN: https://github.com/Homebrew/brew
HEAD: de5a13b94fb26ef95dd11b267c81e79e2372dbbf
Last commit: 4 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 83463db5dcf324d76f4851ed4194809eae25dcc4
Core tap last commit: 54 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit icelake
Clang: 13.0.0 build 1300
Git: 2.30.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.0.1-x86_64
CLT: 13.1.0.0.1.1633545042
Xcode: 13.1


Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  tepl
```

### 

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

I need to use Gedit on my macOS device. It all worked fine with macOS Big Sur version but after the latest update "Monterey" I can't use it. 

### What happened (include all command output)?

(gedit:77777): Pango-WARNING **: 00:09:37.063: couldn't load font "Courier Bold Condensed Not-Rotated 20", modified variant/weight/stretch as fallback, expect ugly output.

### What did you expect to happen?

It should let me use Gedit but I can't.

### Step-by-step reproduction instructions (by running `brew` commands)

Gedit

(gedit:77777): Pango-WARNING **: 00:09:37.063: couldn't load font "Courier Bold Condensed Not-Rotated 20", modified variant/weight/stretch as fallback, expect ugly output.

Brew doctor

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  tepl

**Comment by carlocab** [comment]
Tepl is no longer maintained, so I don't think we can expect it to work properly. Does Gedit work without Tepl?

**Comment by Baccio26** [comment]
No, it doesn’t work. Using brew doctor tepl was the only dependencies that
wasn’t functioning so I assumed it was the problem.
Once I run Gedit it gives me te Pango Warning I linked and then it seems
like there is some visualisation issue because I can only see the upper bar
(the one in which you can see the name of the file and application) and not
the part where I should actually write.
Before updating macOS it worked.
Il giorno gio 4 nov 2021 alle 04:01 Carlo Cabrera ***@***.***>
ha scritto:

**Comment by carlocab** [comment]
This sounds like something only the Gedit developers know how to fix. Please report this to them.

**Comment by Baccio26** [comment]
Ok thanks, I'll do that. Do you know how can I reach out to them?

Il giorno gio 4 nov 2021 alle 08:15 Carlo Cabrera ***@***.***>
ha scritto:

**Comment by carlocab** [comment]
Not really, but I assume you can find this information in the homepage: `brew home gedit`

**Comment by Baccio26** [comment]
Thanks, I'll check it out.

Il giorno gio 4 nov 2021 alle 08:20 Carlo Cabrera ***@***.***>
ha scritto:

**Comment by giuspen** [comment]
This is not a Gedit issue but a GTK3 issue. I'm having the same problem with Cherrytree https://github.com/giuspen/cherrytree/issues/1844

**Comment by carlocab** [comment]
Ah, that'll be this problem then: https://gitlab.gnome.org/GNOME/gtk/-/issues/4342

**Comment by giuspen** [comment]
Thanks @carlocab I'll link the issue you pointed to, hopefully the next Gtk3 release will come soon

**Comment by fxcoudert** [comment]
I have a fix incoming for that, will submit a PR today.

**Comment by carlocab** [comment]
Closed in ba2f61b21aa254147ad02f95465e81b5c60cdfe2.
